x/tokenregistry/keeper/migrations.go

```diff
@@ -24,19 +24,3 @@ func (m Migrator) MigrateToVer2(ctx sdk.Context) error {
 	m.keeper.SetRegistry(ctx, registry)
 	return nil
 }
-
```

Comment:
Will we need a migration here to remove old admin state?

Reply:
Done.